app_server/test/models/User.js

```diff
@@ -272,10 +272,8 @@ lab.experiment('User Model Tests', function () {
       {username: 'Jane', password: 'asdf', email: 'jane@gmail.com'},
       {username: 'Alan', password: 'asdf', email: 'alan@gmail.com'},
       {username: 'John', password: 'asdf', email: 'john@gmail.com'}
-    ])
-    .then(function() {
-      Storage.getListOfUsers()
-      .then(function(users) {
+    ]).then(function() {
+      Storage.getListOfUsers({order: 'asc'}).then(function(users) {
```

Comment:
`]).then(() => Storage.getListOfUsers({order: 'asc'})).then(function(users) {`?
I don't think there's any need to nest a promise.

Looks okay to merge after that.